hw5-10_to_do_list/src/ts/Repository.ts

```diff
@@ -0,0 +1,9 @@
+export type EntityID = number
+
+export interface Repository<T> {
+    getAll(): Array<T> | Promise<Array<T>>
+    add(entity: T): T | Promise<T>
```

Comment:
Just `Promise<T>` and in other places, too
```suggestion
    add(entity: T): Promise<T>
```